src/lib/django-activities/activities/notifiers/oauth/base.py

```diff
@@ -74,7 +74,7 @@ def get_credentials(cls, filename):
         The return value is a dictionary to initialize a
         requests_oauthlib.OAuth1
         """
-        if not os.path.exists(filename):
+        if not os.path.exists(filename) and not settings.TESTING:
```

Comment:
I'd like to suppress warnings on the testing environment.
However, this change is not working.

Because `setting.TESTING` seems to be set after loading this activity notifier.
Could you fix it?